src/core/chart.js

```diff
@@ -0,0 +1,252 @@
+import _ from 'underscore';
+
+
+var PRECISION = 2;
+
+/* This function takes one or more JSON objects from the 
+ * "timeseries" API call with this format:
+ * 
+ * {
+ * "id": "tasmax_mClim_BCCAQv2_bcc-csm1-1-m_historical-rcp45_r1i1p1_20700101-20991231_Canada",
+ * "units": "degC",
+ * "data": {
+ *   "2085-01-15T00:00:00Z": -17.498223073165622,
+ *   "2085-02-15T00:00:00Z": -15.54878007851129,
+ *   "2085-03-15T00:00:00Z": -11.671093808333737,
+ *                    ...
+ *    }
+ * }
+ * 
+ * and return a C3 graph object displaying them.
+ * 
+ * It takes an arbitrary number of data objects, but no more than
+ * two separate unit types. Allowable data resolutions are monthly(12), 
+ * seasonal (4), or yearly (1); an error will be thrown 
+ * if this function is called on data with another time resolution.
+ */
+var timeseriesToAnnualCycleGraph = function(metadata, ...data) {
```

Comment:
Just a little aside: Is there some reason we prefer this `var f = function(args) ...` style declaration to the simpler `function f(args) ...` style?